cxx-qt-build/src/lib.rs

```diff
@@ -61,7 +61,10 @@ impl GeneratedCpp {
         let mut cxx_qt = None;
         // TODO: later change how the resultant filename is chosen, can we match the input file like
         // CXX does?
-        let mut module_ident: String = "".to_owned();
+        //
+        // For CXX-Qt generating one header per QObject likely makes sense, but what happens with CXX data?
+        // for now this uses the module ident
+        let mut file_ident: String = "".to_owned();
```

Comment:
We might need this to be at least one CXX-Qt header per module block (or per file) and not per QObject. Otherwise where does the include line go in a block like this.
```rust
#[cxx_qt::bridge] 
mod ffi {
  #[cxx_qt::qobject(base = "QAbstractListModel")]
  struct MyObject;

  #[cxx_qt::qobject]
  struct AnotherObject;

  extern "C++" {
    include!("<QAbstractListModel>");
  }
}
```

Reply:
So the problem if we use the rust file name is, how do we generate `include!("cxx-qt-gen/include/my_object.cxxqt.h");` in the Rust macro ? as we are inside another macro how can we find the file name ?

Reply:
Wonder if we have `cxx_qt_include!("<QAbstractListModel>")`, any `cxx_qt_include` can go into the .cxxqt.h and any `include` can go into the .cxx.h

Reply:
We could use the nightly feature to get the source file of the macro https://doc.rust-lang.org/proc_macro/struct.SourceFile.html

Reply:
Moved the source file changes into #200. For now this pull request uses the QObject ident name as the generated name.